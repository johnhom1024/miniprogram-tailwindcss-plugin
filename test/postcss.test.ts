import { describe, it, expect } from "vitest";
import { getTwCss } from '@test/helpers/getTwCss';

describe("postcss plugin", () => {

  it("base utitlities output", async () => {
    const res = await getTwCss('<view class="h-[10px] w-[20px] bg-[rgba(255,254,253,.5)]"></view>');
    expect(res.css).toMatchSnapshot();
  });
});
