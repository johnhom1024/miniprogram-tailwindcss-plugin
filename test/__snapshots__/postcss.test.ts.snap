// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`postcss plugin > base utitlities output 1`] = `
".h-\\[10px\\] {
    height: 10px
}
.w-\\[20px\\] {
    width: 20px
}
.bg-\\[rgba\\(255\\2c 254\\2c 253\\2c \\.5\\)\\] {
    background-color: rgba(255,254,253,.5)
}"
`;
